mage_ai/server/utils/output_display.py

```diff
@@ -113,6 +114,27 @@ def __custom_output():
         _shape = _internal_output_return.shape
         _index = _sample.index.tolist()
 
+        _json_string = simplejson.dumps(
+            dict(
+                data=dict(
+                    columns=_columns,
+                    index=_index,
+                    rows=_rows,
+                    shape=_shape,
+                ),
+                type='table',
+            ),
+            default=encode_complex,
+            ignore_nan=True,
+        )
+        return print(f'[__internal_output__]{{_json_string}}')
+    elif isinstance(_internal_output_return, pyspark.sql.DataFrame):
```

Comment:
You can do `type(df).__module__ == 'pyspark.sql.dataframe'` so that you don't need to import pyspark